src/main/java/com/softserveinc/ita/nromanchuk/GoogleHomePage.java

```diff
@@ -1,20 +0,0 @@
-package com.softserveinc.ita.nromanchuk;
```

Comment:
DO NOT CHANGE someone else's code

Reply:
I accidentally did it and did not notice. I will be more attentive

Reply:
I accidentally did it and did not catch sight of. I will be more attentive